src/main/java/uk/gov/register/resources/v1/ItemResource.java

```diff
@@ -1,15 +1,52 @@
 package uk.gov.register.resources.v1;
 
+import com.codahale.metrics.annotation.Timed;
+import uk.gov.register.core.EntryType;
+import uk.gov.register.core.Item;
 import uk.gov.register.core.RegisterReadOnly;
+import uk.gov.register.exceptions.FieldConversionException;
+import uk.gov.register.resources.v2.BlobResource;
+import uk.gov.register.service.ItemConverter;
+import uk.gov.register.views.AttributionView;
+import uk.gov.register.views.ItemListView;
+import uk.gov.register.views.ItemView;
 import uk.gov.register.views.ViewFactory;
+import uk.gov.register.views.representations.ExtraMediaType;
 
 import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import java.util.Collection;
+import java.util.stream.Collectors;
 
 @Path("/items")
```

Comment:
I think it may be prudent to remove or comment out this route until it is feature complete to include in V1.

Reply:
Will do